daemon/containerd/snapshotters_test.go

```diff
@@ -0,0 +1,58 @@
+package containerd
+
+import (
+	"testing"
+
+	"github.com/containerd/containerd"
+	"gotest.tools/v3/assert"
+)
+
+func TestSnapshotterFromGraphDriver(t *testing.T) {
```

Comment:
I wonder if this test has any positive value anymore - it made more sense when there was the logic with stripping the fully qualified name (io.containerd.snapshotter.v1).
Now when the function under test is really just one simple switch-case the test is basically a more verbose duplicate of the function that checks if Go's switch-case works as expected 😄

Reply:
Yes, it's quite a basic test now; I guess it could still be somewhat useful to test the behaviour for defaults and "other" storage drivers (just in case someone makes changes)